common/constants.js

```diff
@@ -63,7 +63,8 @@ export const font = {
 };
 
 export const typescale = {
-  lvl0: `0.8rem`,
+  lvlN1: `12px`,
```

Comment:
I think we can come up with better naming than this, Ideally I'd like to not downsize fonts so much but I can see the point. Why not try a value like `0.8rem` and use a converter here

http://www.standardista.com/px-to-rem-conversion-if-root-font-size-is-16px/

Reply:
ok, made the convert, and lmk if you think we should increase the size a bit.